### ISSUE TITLE
IE - Unable to get property `length` of undefined or null reference

### ISSUE THREAD

**Issue by thisandagain** [LLK/scratch-render#85]
### Steps to Reproduce

- Load VM playground in Microsoft Edge

### Reference
```
SCRIPT5007: Unable to get property 'length' of undefined or null reference
scratch-render.js (11347,1)
```

```
if (this._svgDom.children.length < 1 || this._svgDom.children[0].localName !== 'svg') {
    throw new Error('Document does not appear to be SVG.');
}
```

### Operating System and Browser
Windows 10  |  Edge (Latest)

**Comment by TheBrokenRail** [comment]
Fixed with #100

**Comment by rschamp** [comment]
Thanks @TheBrokenRail!
